### ISSUE TITLE
parts list

### ISSUE THREAD

**Issue by dirkjanfaber** [dchote/robot-mower#1]
This might become a very interesting project, that I would like to follow and participate in. Is there a part list of the used hardware? And can you add the stl files/links for the 3d printed parts?

**Comment by dchote** [comment]
Hey. Once I get the first functional prototype I’ll be publishing the STLs and a parts list. I didn’t want people running off and spending money until I had something that works 😉

**Comment by dirkjanfaber** [comment]
Makes sense. Anyway, I'm gonna follow this project and hope to contribute to it.
